README.fr.md

```diff
@@ -25,9 +25,13 @@ Upsilon est un fork d'Omega, un fork d'Epsilon, l'OS de Numworks tournant sur le
 
 ## Installation
 
-### Manuelle
+### Installeur
+rendez-vous sur le [ site d'Upsilon](https://lolocomotive.github.io/Upsilon-website) puis rendez-vous à la section "Installer". 
+Vous trouverez dans celle-ci l'installeur, qui, si votre calculatrice est reconnue, ne contient pas Epsilon 16 et que votre navigateur accepte WebUSB, vous proposera d'installer Upsilon.  
+Une fois le protocole d'installation lancé, il est important que vous ne débranchiez pas votre calculatrice.  
+Quand l'installeur vous signalera qu'Upsilon a été installé avec succès, vous pouvez débrancher votre calculatrice.
```

Comment:
No, I don't think. It might look like I'm being uselessly untrustfull towards the user, ut one of the things I know, is that you must always assume that at least one person is  going to do something dumb if not instructed not to.

Reply:
But maybe the formulation can be simplified.
```suggestion
Ne débranchez votre calculatrice qu'une fois l'installation terminée.
```

Reply:
OK, seems fine. Although I don't agree at 100% on the idea let's proceed with this, but then I suggest that you suggest similar changes to the english Readme. I'll let you do it so it can be exactly the way you want.